src/kudu/consensus/raft_consensus.cc

```diff
@@ -1346,6 +1352,14 @@ static bool IsConsensusOnlyOperation(OperationType op_type) {
   return op_type == NO_OP || op_type == CHANGE_CONFIG_OP;
 }
 
+Status RaftConsensus::StartFollowerTransactionUnlocked(
+    const ReplicateMsgWrapper& msg_wrapper) {
+  if (!msg_wrapper.GetUncompressedMsg()) {
+    return Status::IllegalState("Rejected: Msg wrapper is null");
+  }
+  return StartFollowerTransactionUnlocked(msg_wrapper.GetUncompressedMsg());
```

Comment:
Are you adding the checksum check on the fbcode side?

Reply:
Yep